pkg/workload/schemachange/operation_generator.go

```diff
@@ -1089,6 +1089,10 @@ func (og *operationGenerator) createIndex(ctx context.Context, tx pgx.Tx) (*opSt
 	duplicateRegionColumn := false
 	nonIndexableType := false
 	def.Columns = make(tree.IndexElemList, 1+og.randIntn(len(columnNames)))
+	jsonInvertedIndexesAllowed, err := isClusterVersionLessThan(ctx, tx, clusterversion.ByKey(clusterversion.V23_2))
```

Comment:
this seems backwards -- the check is `version < 23.2` but the variable name is `jsonInvertedIndexesAllowed`. it should either be:
```
jsonInvertedIndexesAllowed, err := isClusterVersionGreaterThan(ctx, tx, clusterversion.ByKey(clusterversion.V23_2))
```
or
```
jsonInvertedIndexesNotSupported, err := isClusterVersionLessThan(ctx, tx, clusterversion.ByKey(clusterversion.V23_2))
```

Reply:
Ah, good catch